addons/interaction/functions/fnc_getBackpackPos.sqf

```diff
@@ -0,0 +1,22 @@
+#include "..\script_component.hpp"
+/*
+ * Author: DartRuffian
+ * Return a suitable position for the action point for the target's backpack.
+ *
+ * Arguments:
+ * None (uses local variable _target)
+ *
+ * Return Value:
+ * Position of _target's backpack in model space <ARRAY>
+ *
+ * Example:
+ * call ace_interaction_fnc_getBackpackPos
+ *
+ * Public: No
+ */
+
+//IGNORE_PRIVATE_WARNING ["_target"];
+
+private _position = _target selectionPosition ["spine3", "Memory"];
+_position = _position vectorAdd [0, -0.2, 0.05];
```

Comment:
This works for most actual backpacks, but there are vanilla backpacks that are actually leg pouches. Maybe this should be used as a default and allow it to be overwritten in the backpack's config?

Reply:
https://github.com/johnb432/AIMEE/blob/master/addons/inventory/functions/fnc_backpackPos.sqf is what I use to determine the position, which *should* work for different models of characters (I believe, never tried it).

Reply:
Doesn't look like backpacks were properly set up for it, the backpack just appears at the model's center.

The actual position for the action ends up being around the chest. If they ever added the ability for it to wear backpacks, they'd need to overwrite the `position` property.